src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -40,43 +50,50 @@
   @Override
   public void robotInit() {
 
+    //Cameras
+    stream = new Stream();
+    //CameraServer.getInstance().startAutomaticCapture();
+    stream.streamCamera();
+    grip = new GripPipeline();
+    
+    //Compressor
     c = new Compressor(0);        //Initialize Compressor
     c.setClosedLoopControl(true); // Start Compressor in Closed Loop Control
 
-
-    stream.streamCamera();
-    // GripPipeline grip = new GripPipeline();
-
     // Subsystems
     powerDistributionPanel = new PowerDistributionPanel();
     drivetrainSubsystem = new DriveTrainSubsystem();
     intakeSubsystem = new IntakeSubsystem();
     elevatorSubsystem = new ElevatorSubsystem();
+    elevatorArmSubsystem = new ElevatorArmSubsystem();
 
     // Sensors
-    lidar = new LidarLitePWM(4);
+    lidar = new LidarLitePWM(RobotMap.lidarPort);
 
     // Operator Input
+    
 
-    driverGamepad = new BionicF310(RobotConstants.driverGamepadPort, // Port
+
+    driverGamepad = new BionicF310(RobotMap.driverGamepadPort, // Port
                                    RobotConstants.driverGamepadDeadzone, // Deadzone
                                    RobotConstants.driverGamepadSensitivity // Gamepad sensitivity
     );
 
-    manipulatorGamepad = new  BionicF310(RobotConstants.manipulatorGamepadPort, // Port
+    manipulatorGamepad = new BionicF310(RobotMap.manipulatorGamepadPort, // Port
                                          RobotConstants.manipulatorGamepadDeadzone, // Deadzone 
                                          RobotConstants.manipulatorGamepadSensitivity // Gamepad sensitivity
     );
-
+    manipulatorGamepad.buttonHeld(BionicF310.RT, 0.2, new CargoIntakeIn());
+    manipulatorGamepad.buttonHeld(BionicF310.RB, new CargoIntakeOut());
+    manipulatorGamepad.buttonHeld(BionicF310.LT, 0.2, new HatchPanelIntakeOpen());
+    
     driverGamepad.buttonPressed(BionicF310.A, new Linefollow());
-    driverGamepad.buttonPressed(BionicF310.X, new CargoIntakeIn());
-    driverGamepad.buttonPressed(BionicF310.Y, new CargoIntakeOut());
-    driverGamepad.buttonPressed(BionicF310.LB, new HatchPanelIntakeOpen());
-    driverGamepad.buttonPressed(BionicF310.RB, new HatchPanelIntakeClose());
-    driverGamepad.buttonPressed(BionicF310.Start, new InvertDriveDirection());
+    driverGamepad.buttonPressed(BionicF310.RT, 0.2, new InvertDriveDirection());
+
+    manipulatorGamepad.buttonPressed(BionicF310.A, new SetElevatorPosition(-27220,1));
```

Comment:
all of the numbers used in lines 86-93 should preferably in constants or at least commented well, because it's not very apparent at the moment